### PR TITLE
feat: landing page components

### LINKED ISSUE #33: Landing Page components

### PR DESCRIPTION
**Pull request by MohammedMahani**

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -5,12 +5,12 @@ import {
...
 } from 'react-router-dom';
 
-import { Signup } from './pages';
+import { Signup, LandingPage } from './pages';
 
 const router = createBrowserRouter([
   {
     path: '/',
-    element: <div>Hello landing</div>,
+    element: <LandingPage/ >,
```
Comment (abdullah-alsharif) [inline client/src/App.tsx]: you can use home or anything else for this router 
Landingpage is tall and not common

File: client/src/components/LandingPage/Feature.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        </Col>
+      </Row>
+    </div>
+  );
+}
+
+export default Feature;
+
+
+{/* <div className="feat-sec">
```
Comment (abdullah-alsharif) [inline client/src/components/LandingPage/Feature.tsx]: remove the comment

File: client/src/components/LandingPage/AboutUs.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+function AboutUs() {
+  return (
+    <div className="aboutus-sec">
+      <Row>
+        <Col span={24}>
+
+          <div className="about-title">
+            <h3>
+              <strong>All-In-One</strong>
+              {' '}
```
Comment (devstm) [inline client/src/components/LandingPage/AboutUs.tsx]: what is all in one and why you need me to merge this 'Lorem, ipsum' in main branch?

File: client/src/components/LandingPage/AboutUs.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dignissimos,
+              reprehenderit reiciendis nulla magnam
+            </p>
+          </div>
+        </Col>
+      </Row>
+
+      <Row>
+        <Col span={8}>
+          <Card className='gg'>
```
Comment (devstm) [inline client/src/components/LandingPage/AboutUs.tsx]: what is card content ?

File: client/src/components/LandingPage/AboutUs.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            <p>Card content</p>
+            <p>Card content</p>
+            <p>Card content</p>
+          </Card>
+        </Col>
+        <Col span={8}>
```
Comment (devstm) [inline client/src/components/LandingPage/AboutUs.tsx]: dont repeat the code use array.map and split the code into new cmponents

File: client/src/components/LandingPage/Feature.tsx [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+    <div className="feat-sec">
+
+      <div className="feature-title">
+        <h3>
+          Our
+          {' '}
+          <strong>Features</strong>
+        </h3>
+        <p>
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dignissimos,
```
Comment (devstm) [inline client/src/components/LandingPage/Feature.tsx]: Be more creative in choosing words

File: client/src/components/LandingPage/Feature.tsx [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+        </p>
+      </div>
+
+      <Row className='feat-container'>
+        <Col md={12}>
+          <Image
```
Comment (devstm) [inline client/src/components/LandingPage/Feature.tsx]: each image should has alt and fallback attributes

File: client/src/components/LandingPage/Feature.tsx [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+            className='right-image'
+            src={secFeat}
+            preview={false}
+          />
+        </Col>
+      </Row>
+
+      <Row className='feat-container'>
```
Comment (devstm) [inline client/src/components/LandingPage/Feature.tsx]: I think this code is repeated more than once

File: client/src/components/LandingPage/Hero.tsx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  Image,
+  Button,
+} from 'antd';
+import heroImage from '../../assets/img/hero.png';
+import './LandingPage.css'
+
+function Hero() {
+  return (
+    <div className="hero-sec">
+      <Row gutter={[8, 8]} style={{ marginRight: 0 }}>
```
Comment (devstm) [inline client/src/components/LandingPage/Hero.tsx]: we use inline stile just for test not to merge it into the main so dont push any inline stilyle on Github

File: client/src/components/LandingPage/LandingPage.css
Comment (devstm) [inline client/src/components/LandingPage/LandingPage.css:1]: rename this file to style.module.css

File: client/src/components/common/Footer.tsx
Comment (devstm) [inline client/src/components/common/Footer.tsx:1]: Is that all the footer will contain?
I looked at the figma and I think this component is very small compared to the size of the footer there -in figma-

File: client/src/components/common/NavBar.tsx [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+function Nav() {
+  return (
+    <div className="nav-bar">
+      <Row align="middle">
+        <Col span={8}>
+          <div className="logo">Logo</div>
+        </Col>
+        <Col span={8}>
+          <Menu mode="horizontal" defaultSelectedKeys={['1']} style={{ background: 'none', display: 'flex', alignItems: 'center' }}>
```
Comment (devstm) [inline client/src/components/common/NavBar.tsx]: inline style

**Review by devstm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**